test/ci-scripts/Write-TestResults.ps1

```diff
@@ -527,4 +567,4 @@ Snippet that will be placed in the README.md files
 <IMG SRC="https://azurequickstartsservice.blob.core.windows.net/badges/100-blank-template/BestPracticeResult.svg" />&nbsp;
 <IMG SRC="https://azurequickstartsservice.blob.core.windows.net/badges/100-blank-template/CredScanResult.svg" />&nbsp;
```

Comment:
I think you need to update the snippet that gets pushed to the README files with the new badge.

Reply:
+1

Reply:
We will need to, though I don't think we'll do this (and all the readme files) until we GA and figure out the TTK story.  So we don't have almost 2 identical badges